estela-web/src/components/JobDetailPage/index.tsx

```diff
@@ -787,57 +807,68 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
-                                        cy="144"
-                                        r={`${radius}`}
-                                        stroke="currentColor"
-                                        strokeWidth="5"
-                                        fill="transparent"
-                                        strokeDasharray={circunference}
-                                        strokeDashoffset={circunference - storagePercentage * circunference}
-                                        className="text-estela-states-green-medium"
-                                    />
-                                </svg>
-                                <Content className="absolute items-center justify-center">
-                                    <span className="text-3xl text-center">{`${storage.quantity}${storage.type}`}</span>
-                                    <br />
-                                    <span className="text-lg text-center mx-5">of 1GB</span>
-                                </Content>
+                                <Doughnut
+                                    plugins={[
+                                        {
+                                            id: "storageNeedle",
+                                            afterDatasetDraw(chart) {
+                                                const { ctx } = chart;
+                                                ctx.save();
+                                                const x = chart.getDatasetMeta(0).data[0].x;
+                                                const y = chart.getDatasetMeta(0).data[0].y;
+                                                ctx.textAlign = "center";
+                                                ctx.textBaseline = "middle";
+                                                ctx.font = "1.875rem/2.25rem sans-serif";
+                                                ctx.fillText(`${storage.quantity} ${storage.type}`, x, y - 20);
+                                                ctx.font = "1.25rem/1.75rem sans-serif";
+                                                ctx.fillText("of 1GB", x, y + 20);
+                                            },
+                                        },
+                                    ]}
+                                    options={{
+                                        responsive: true,
+                                        events: [],
+                                    }}
+                                    data={{
+                                        datasets: [
+                                            {
+                                                label: "GB",
+                                                data: [...dataChartProportions],
+                                                backgroundColor: [...colorChartArray],
+                                                borderWidth: 1,
+                                                cutout: "90%",
```

Comment:
This line shows error

Reply:
The error is in the cutout property, that is not present on the `data` prop of the `Doughnut` component, this is a lint error of [(react-chartjs-2.js)](https://react-chartjs-2.js.org/) library because it was created as a wrapper for chartjs v2 library ([Why does react-chartjs-2 have "2" in its name?](https://react-chartjs-2.js.org/faq/why-two)) and the data prop does not have this `cutout` property. Anyway don't worry many features of chartjs 3 are still rendered because react-chartjs is only a wrapper of the chartjs3 library, cutout property is still considered by chartjs to be rendered.

Reply:
There is no way to workaround this error? When you set `datasets` outside from `data` that error doesn't appear. Anyway keep that approach